### PR TITLE
Define goals

### LINKED ISSUE #2: Goals
The goals are:
- stateless chain
- wasm native execution: this should focus on linking code with libraries as opposed to "calling" other libraries
- currency agnostic

### PR DESCRIPTION
**Pull request by axic**
Closes #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Wortels
-This is the design repo for wortels, a specification for some random project.
+
+This is the design repo for Wortels - a novel design for a consensus computer.
```
Comment (jakelang) [inline README.md]: maybe "specification for a stateless WebAssembly-based blockchain computer."

File: README.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+It preliminary takes learnings from Ethereum.
```
Comment (jakelang) [inline README.md]: Not sure what this sentence means.
Maybe: "It is intended to build on learnings from Ethereum, an existing blockchain computer."

File: README.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+
+The goals are:
+- stateless chain
```
Comment (jakelang) [inline README.md]: `- Stateless chain`
capitalization

File: README.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+This is the design repo for Wortels - a novel design for a consensus computer.
+
+It preliminary takes learnings from Ethereum.
+
+The goals are:
+- can operate as a pegged chain or sidechain
+- stateless chain
+- no native token (aka "account abstraction")
+- WebAssembly as an execution engine (this should focus in linking code)
+- no compatibility with EVM is strived for by the design
```
Comment (jakelang) [inline README.md]: Maybe this should not be a design goal. It instead could be noted as
"Compatibility with Ethereum's EVM is not a primary goal of the project."

File: README.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
...
+
+The basic data structure for the tree is [explained here](./trie.md).
```
Comment (jakelang) [inline README.md]: I think we are using the spelling "trie"

Comment (axic) [reply README.md]: I like "tree" because it is general. "trie" is a specific one.

Comment (jakelang) [reply README.md]: right, nvm

File: README.md [hunk 6]
```diff
@@ -1,5 +1,18 @@
 # Wortels
-This is the design repo for wortels, a specification for some random project.
+
+This is the design repo for Wortels - a novel design for a stateless WebAssembly-based consensus computer.
+
+It is intended to build on learnings from Ethereum, an existing blockchain computer.
+
+The goals are:
+- Can operate as a pegged chain or sidechain
```
Comment (jakelang) [inline README.md]: Not to be picky but `To operate` may look nicer here.

File: README.md [hunk 7]
```diff
@@ -1,5 +1,18 @@
...
+- Stateless chain
+- No native token (aka "account abstraction")
+- WebAssembly as an execution engine (this should focus in linking code)
```
Comment (jakelang) [inline README.md]: and `this should focus on`

File: README.md [hunk 8]
```diff
@@ -1,5 +1,18 @@
...
+- Compatibility with Ethereum's EVM is not a primary goal
```
Comment (jakelang) [inline README.md]: Maybe not include `this is not a goal` as a goal, alternatively note it elsewhere in the doc.

**Pull request merged**